lib/routes.php

```diff
@@ -167,6 +167,7 @@
 $app->post('/playlist/widget/:type/:id', '\Xibo\Controller\Module:addWidget')->name('module.widget.add');
 $app->put('/playlist/widget/:id', '\Xibo\Controller\Module:editWidget')->name('module.widget.edit');
 $app->delete('/playlist/widget/:id', '\Xibo\Controller\Module:deleteWidget')->name('module.widget.delete');
+$app->delete('/playlist/dashboard/widget/:id', '\Xibo\Controller\Module:deletePlaylistWidget')->name('playlist.module.widget.delete');
```

Comment:
I dont see this method added to the Module controller?
Even so - perhaps it would be better to have this in the Playlist Dashboard controller, and for the URL to be playlistdashboard?